pkg/config/config.go

```diff
@@ -122,8 +122,8 @@ func NewConfigBundle(opts ...BundleOption) (*v1alpha1.ConfigBundle, error) {
 	return bundle, nil
 }
 
-// New initializes and returns a Configurator.
-func New(c Content) (config runtime.Configurator, err error) {
+// newConfig initializes and returns a Configurator.
```

Comment:
went ahead and made this function, `fromFile`, and `fromBytes` private since I removed all references to them and replaced with the proper `NewConfigFrom*` function.

Reply:
nit: The package is already called "`config`". The extra "`Config`" in the `NewConfigFrom*` functions is redundant.